legacy_tester/parsed_json_test_definitions.py

```diff
@@ -3,5 +3,26 @@ def phase_one_regions_parse_not_included_countries(published_json, test_payload)
     for to_be_excluded_countries in test_payload:
         if to_be_excluded_countries in parsed_countries:
             print('Error: {} found in list of phase 1 parsed countries'.format(to_be_excluded_countries))
-            return
-    print("PASS")
\ No newline at end of file
+            return False
+    return True
+
+
+def unit_of_subtype_of_country_should_contain_exactly_x_tonals_remarks_ending_with_y(published_json, test_payload):
+    for unit_payload in test_payload:
+        country_id_to_check = next(filter(lambda a: a.country == unit_payload['country'],
+                                          published_json['countries'])).id
+        subtype_id_to_check = next(filter(lambda a: a.platform_sub_type == unit_payload['sub'],
+                                          published_json['platform_sub_types'])).id
+        unit_id_to_check = next(filter(lambda a: a.title == unit_payload['unit'] and
+                                                 a.platform_sub_type_id == subtype_id_to_check and
+                                                 a.country_id == country_id_to_check,
+                                       published_json['units'])).id
+        tonals_to_check = filter(lambda a: a.unit_id == unit_id_to_check, published_json['tonals'])
+        tonals_with_required_remarks = filter(lambda a: a.remarks.endswith(unit_payload['remarks_suffix']),
```

Comment:
can we have `includes` rather than `endswith` please?

Reply:
Done. There's no `included` or `contains`. Have used `find` with return index instead.